definitions/divorce/json/CaseEventToFields/CaseEventToFields-bailiff-nonprod.json

```diff
@@ -114,5 +114,19 @@
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "22/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorApplyForDN",
+    "CaseFieldID": "LabelContinueBecauseOfSuccessfulBailiff",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "READONLY",
+    "PageID": "dnreviewaos",
+    "PageLabel": "Review Acknowledgement of Service",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "SuccessfulServedByBailiff=\"Yes\" AND respondentanswerslink !=\"*\"",
```

Comment:
can you please make sure all roles that can trigger `solicitorApplyForDN` event have `CRU` for these fields?